client/my-sites/site-settings/section-export.jsx

```diff
@@ -7,10 +7,25 @@ import { Provider } from 'react-redux';
 /**
  * Internal dependencies
  */
+import EmptyContent from 'components/empty-content';
 import ExporterContainer from 'my-sites/exporter';
+import i18n from 'lib/mixins/i18n';
 
 export default class SiteSettingsExport extends Component {
 	render() {
+		if ( this.props.site.jetpack ) {
+			return (
+				<EmptyContent
+					illustration="/calypso/images/drake/drake-jetpack.svg"
+					title={ i18n.translate( 'Want to export your site?' ) }
+					line={ i18n.translate( 'Visit your site\'s wp-admin for all your import and export needs.' ) }
```

Comment:
I'm not 100% sure what standards dictate here, but I think that in general we prefer to see `"Visit your site's"` vs. `'Visit your site\'s'` when there's a quote that'll need to be escaped, since it's a little easier on translators to not have to think about the escaping.